### DISCUSSION TITLE
Cannot get images to work with page bundles.

### DISCUSSION THREAD

**Discussion by georgew509** [chipzoller/hugo-clarity#281] [Q&A] [answered]
I have not been able to get images working with "usePageBundles = true" for pages or posts.

For example I have:

content/
  posts/
   2022/
     pair-networks-vps/
        index.md
        images/cloud_server.png

And in index.md I have:

+++
title = "Pair Networks VPS"
date = 2022-01-08
tags = ["Pair Networks", "Shared Hosting", "VPS"]
draft = false
description = "I just moved from a Pair Networks shared host to a Pair Networks VPS Starter II host. I am writing a brief note about the move and my experience."
usePageBundles = true
+++

![Image by Jorge Guillen from Pixabay](images/cloud_server.png)

I start up the Hugo server and all I see is a blank box with a ?mark in it and the Alt text.  If I set "usePageBundles" to false, then I see the debug message about image not found.

With usePageBundles set to true, I looked at the generated html code, and it's trying to load the image from the root directory:

<img class="image_figure image_internal" src="https://altoplace.org/images/cloud_server.png" alt="Image by Jorge Guillen from Pixabay" loading="lazy" decoding="async" width="640" height="360"  />

instead of the post directory:

public/posts/2022/pair-networks-vps
$ tree .
.
├── images
│   └── cloud_server.png
└── index.html

Am I doing something wrong?

**Comment by rootwork** [comment, marked as answer]
Hi @georgew509, apologies -- this is a current bug. We should have a fix released soon. #271 may address it if you feel comfortable patching a theme file yourself, although I haven't had a chance to fully test it yet.
